### ISSUE TITLE
Left sidebar w/ high contrast theme

### ISSUE THREAD

**Issue by reiner-text** [nextcloud/server#15694]
### Steps to reproduce
1. Go to the Settings.
2. Check ``Enable high contrast theme`` in the Accessibility section of the Personal Settings. 
3. Open the Files app.

### Expected behaviour
The clarity of the navigation in the left sidebar should be increased.

### Actual behaviour
In the Files app the left sidebar of the main user interface still looks with normal font and/or contrast after enabling the accessibility setting. Only the lower Settings area and the disk usage have a higher contrast.

### Server configuration
**Nextcloud version:** (see Nextcloud admin page)
16.0.1
**Updated from an older Nextcloud or fresh install:**
15.0.7
**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
Firefox 67.0
**Operating system:**
Windows 10 1903

**Comment by reiner-text** [comment]
The same bad readability with 17.0.0 Beta 3.

**Comment by juliushaertl** [comment]
cc @skjnldsv

**Comment by reiner-text** [comment]
> In the Files app the left sidebar of the main user interface still looks with normal font and/or contrast after enabling the accessibility setting.

The same with the sidebar in Personal/Administration Settings.
The sidebar in the users mgmt seems okay.

**Comment by Steve2955** [comment]
I would like to give this a try. All that needs to be done is adding some css selectors in ```server/apps/accessibility/css/themehighcontrast.scss``` right here:

```css
[class^='icon-'], [class*=' icon-'],
.action,
#appmenu li a,
.menutoggle {
	opacity: 1 !important;
}
```

Am I right? Seems like a great first contribution to me.

**Comment by juliushaertl** [comment]
@Steve2955 Yes, that seems to be the right place. When testing your change locally you might need to run `occ maintenance:repair` to make sure the scss cache is cleared and your change applies.

Looking forward to your contribution :partying_face:
